File: ios/Porchivo/Models/IncidentSeverity.swift
//
//  IncidentSeverity.swift
//  Porchivo
//
//  Incident severity values mirroring the DB `incident_severity` enum.
//

import Foundation

enum IncidentSeverity: String, CaseIterable, Identifiable {
    case low
    case medium
    case high
    case critical

    var id: String { rawValue }

    var label: String {
        switch self {
        case .low: return "Low"
        case .medium: return "Medium"
        case .high: return "High"
        case .critical: return "Critical"
        }
    }
}

File: android/app/src/main/java/com/rork/porchivo/model/IncidentSeverity.kt
package com.rork.porchivo.model

/** Incident severity values mirroring the DB `incident_severity` enum. */
enum class IncidentSeverity(val value: String, val label: String) {
    LOW("low", "Low"),
    MEDIUM("medium", "Medium"),
    HIGH("high", "High"),
    CRITICAL("critical", "Critical"),
}
